script/release-flynn

```diff
@@ -116,15 +116,16 @@ main() {
     manifest="$(new_release_manifest)"
   fi
 
-  info "checking for existing release"
-  if jq --exit-status ".versions[] | select(.commit==\"${commit}\")" <<< "${manifest}"; then
-    fail "commit ${commit} has already been released"
-  fi
+  info "getting previous release tag"
+  local previous=$(git tag | tail -n 1)
```

Comment:
This should be explicit about which tags it is listing (i.e. `git tag --list 'v*'`)